[EDU-2445] add examples rte (#396)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/observe/real-time-events/real-time-events.mdx

| $upstream_addr | Client's IP address and port. Can also store multiple servers or server groups. Example: `192.168.1.1:80`. When the response is `127.0.0.1:1666`, the upstream is [Azion Cells Runtime](/en/documentation/products/edge-application/edge-functions/runtime/overview/). |
| $upstream_bytes_received | Number of bytes received by the origin’s edge if the content isn’t cached. Example: `8304` |
| $upstream_bytes_sent | Number of bytes sent to the origin. Example: `2733` |
| $upstream_cache_status | Status of the local edge cache. Can be: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, or `HIT` |
| $upstream_response_time | Time it takes for the edge to receive a default response from the origin in milliseconds, including headers and body. Example: `0.876`. **In case of cache, the response is `-`**. |
| $upstream_status | HTTP status code of the origin. If a server cannot be selected, the variable keeps the 502 (Bad Gateway) status code. Example: `200`. **In case of cache, the response is `-`**. |
| $waf_attack_action | Reports WAF’s action regarding the action. Can be: `$BLOCK`, `$PASS`, `$LEARNING_BLOCK`, or `$LEARNING_PASS` |

> | $waf_attack_family | Informs the attack’s families, which are categories of attack identified by WAF and classified according to the OWASP Top 10 Example: `$XSS`. [See the categories](/en/documentation/products/edge-firewall/web-application-firewall/#main-settings). |

Author: ```suggestion
| $waf_attack_family | Informs the attack’s families, which are categories of attack identified by WAF and classified according to the OWASP Top 10. Example: `$XSS`. [See the categories](/en/documentation/products/edge-firewall/web-application-firewall/#main-settings). |
```

| $waf_block | Informs whether the WAF blocked the action or not. `0` when action wasn’t blocked and `1` when action was blocked. When in *Learning Mode*, it won’t be blocked regardless of the return. |
| $waf_headers | When the request headers sent by the user are analyzed by the WAF module and tagged as **blocked** with `$waf_block = 1`, it contains a `base64` encoded string. Otherwise, it contains a dash character `-`. It applies to both WAF *Learning* or *Blocking* modes. |
| $waf_learning | Informs if WAF is in Learning mode. Can be `0` or `1`. |
| $waf_match | List of infractions found in the end user’s request. It’s formed by key-value elements: the *key* refers to the type of violation detected; the *value* shows the string that generated the infraction. Example: `0:1311:BODY:ctl00_cph_jp1_dados_container_clientstate`. Find out more on [WAF Allowed Rules](/en/documentation/products/edge-firewall/web-application-firewall/#allowed-rules). |

> | $waf_score | Reports the score that will be increased in case of a match with the rules set for the WAF. Example: `0:$SQL:2` |

Reviewer: ```suggestion
| $waf_score | Reports the score that'll be increased in case of a match with the rules set for the WAF. Example: `0:$SQL:2` |
```

| $navigation.typeReserved | Any type of navigation not defined by other specific variables (**navigation.typeNavigate** and **navigation.typeReload**). |
| $networkApi.downlink | The average volume of data received in Mb/s. |
| $networkduration | Duration without query browser waiting. Example: `52` |
| $pageloadtime | Time from the start of navigation until the full page load. Example: `1267` |
| $platform | Operating system architecture. Example: `Linux x86_64`, `Iphone` |
| $redirectcount | Number of redirects since the last navigation without redirection in the context of the current navigation. Example: `0` |
| $referrer | Address of the page the user made the request from. The URL by which the user arrived at “locationHref”. If the access originated directly from the page (for example, through a bookmark), the value will be an `empty string`. It doesn't provide DOM access to the reference page. Example: `https://www.azion.com/pt-br/` |
| $rendertime | The amount of time it took to render the page. Example: `1242` |
| $rtt | Round-Trip Time (RTT) information. Example: `250` |

> | $ssl | Standard protocol for an established TLS connection. Example: `TLS v1.2`. If the requested URL has a secure connection, it returns the time it took for authenticating. Example: `0` |

Author: ```suggestion
| $ssl | Standard protocol for an established TLS connection. Example: `0`. If the requested URL has a secure connection, it returns the time it took for authenticating. |
```

| $tcp | Internet protocol that returns the data that makes up the page. The time it takes for the TCP handshake is the time between the connection start and the connection end. Example: `0` |
| $time | Request date and time. Example: `31 October, 2023, 21:10:55` |
| $timestamp | Request date and time. Example: `31 October, 2023, 21:10:55` |

> | $ttfb | Time To First Byte: time until the arrival of the first byte of the requested page. Example: `1` |

Reviewer: It could be good to add the unit

> | $ttfb | Time To First Byte: time until the arrival of the first byte of the requested page. Example: `1` |

Author: ```suggestion
| $ttfb | Time To First Byte: time until the arrival of the first byte of the requested page in milliseconds. Includes 1 round trip of latency and the time the server took to prepare the response. Example: `1` |
```

| $remote_address | IP address of the origin that generated the request. Example: `54.233.153.15` |
| $server_protocol | The connection established protocol. Example: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
| $time | Request date and time. Example: `31 October, 2023, 21:10:55` |
| $total_blocked | Informs the total number of requests blocked by WAF. Example: `2` |
| $total_processed | Informs the total number of requests processed by WAF. Example: `1` |
| $waf_action | Reports WAF’s action regarding the action. Can be: `$BLOCK`, `$PASS`, `$LEARNING_BLOCK`, or `$LEARNING_PASS`. |
| $waf_args | The request arguments. Example: `quantidade_periodos=10` |
| $waf_learning | Informs if WAF is in Learning mode. Can be `0` or `1` |
| $waf_match | List of infractions found in the end user’s request. It’s formed by key-value elements: the *key* refers to the type of violation detected; the *value* shows the string that generated the infraction. Example: `0:1311:BODY:ctl00_cph_jp1_dados_container_clientstate`. Find out more on [WAF Allowed Rules](/en/documentation/products/edge-firewall/web-application-firewall/#allowed-rules). |

> | $waf_score | Reports the score that will be increased in case of a match with the rules set for the WAF. Example: `0:$SQL:2` |

Reviewer: ```suggestion
| $waf_score | Reports the score that'll be increased in case of a match with the rules set for the WAF. Example: `0:$SQL:2` |
```
